scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
```

Comment:
I don't think this is necessary, you can simply do

```scss
.form-inline-tabs {

}
```
in bootstrap class variations are always applied together with the base classe, like
```scss
<button class="btn btn-primary"></button>
```
so you don't have the selector to be
```scss
.form-inline.form-inline-tabs
```